### DISCUSSION TITLE
Procurando informações

### DISCUSSION THREAD

**Discussion by amozbatera** [blockscout/blockscout#9207] [Support]
Hello how are you, I sent 11 manta to my subwallert wallet and it got stuck like I do on the eth network

**Comment by ekawibisono** [comment]
hello @amozbatera this is blockscout github not manta, you can contact manta network. thanks

**Comment by amozbatera** [comment]
Hello how are you, I sent 11 manta to my subwallert wallet and it got stuck
like I do on the eth network

Em sáb., 20 de jan. de 2024 10:31, Ekacitta Wibisono <
***@***.***> escreveu:

**Comment by ekawibisono** [comment]
hello @amozbatera 
this is blockscout github, Not Manta, you can join grup discord manta https://discord.com/invite/mantanetwork
